setup.cfg

```diff
@@ -23,6 +23,7 @@ install_requires =
     matplotlib >= 2.2.0
     nibabel >= 4.0.1
     nipype >= 1.7.0
+    traits < 6.4
```

Comment:
@effigies should we just cut a new nipype release?

Reply:
Yes, I think so.